src/modules/dataFetching/components/DeleteMutationButton.tsx

```diff
@@ -32,25 +34,24 @@ const DeleteMutationButton = <Mutation, MutationVariables>({
   recordName = 'record',
   onSuccess,
   confirmationDialogContent,
+  verb = 'delete',
 }: DeleteMutationButtonProps<MutationVariables>) => {
   const [showDialog, setShowDialog] = useState(false);
 
   const [deleteRecord, { loading, error }] = useMutation<
     Mutation,
     MutationVariables
-  >(queryDocument, {
-    variables,
-    onCompleted: (result) => {
+  >(queryDocument, { variables });
+  const handleDelete = useCallback(() => {
+    deleteRecord().then((result) => {
+      // fixme: should probably look for errors[] in mutation response
```

Comment:
Yeah, probably. I can update it to use the same `ErrorState` pattern we have elsewhere. This also doesn't work with  warnings (errors where `severity: warning`), which we will probably want at some point.